lib/oban/queue/executor.ex

```diff
@@ -170,10 +170,16 @@ defmodule Oban.Queue.Executor do
   def emit_event(%__MODULE__{state: :failure} = exec) do
     measurements = %{duration: exec.duration, queue_time: exec.queue_time}
 
+    kind =
+      case exec.kind do
```

Comment:
Initially, I wasn't sure about this. Now that I see it, I appreciate the explicitness. Nice work 👍